aten/src/ATen/native/native_functions.yaml

```diff
@@ -6702,19 +6702,19 @@
 - func: _to_cpu(Tensor[] tensors) -> Tensor[]
   variants: function
 
-- func: to_dense(Tensor self, ScalarType? dtype=None) -> Tensor
+- func: to_dense(Tensor self, ScalarType? dtype=None, bool? masked=False) -> Tensor
   variants: method
```

Comment:
So, we are fine with making bc-breaking changes right now?

Reply:
In fact, the previous behavior for `to_dense` backward to ignore the unspecified elements (the `masked==True` case) was incorrect for all `torch` operations (`torch.sum`, `torch.mm`, etc) that support sparse returns.

Reply:
OK, I could clean up some gradchecks then once `sparse_mask` is differentiable.

Reply:
UPDATE: the default `masked` is `True` for BC as agreed in the 3/7/2023 meeting.